/**:
  ros__parameters:

    image_transport: "compressed"

    num_streams: 1

    stream0: [ "/arena/cam1/image", "/arena/cam1/camera_info", "map" ]
    stream0_offset: [ 0., 0., 0.8, 0.653, -0.653, 0.271, -0.271 ]

    feature:  # detection export key: 0=off, 1=enable, -1=only when filtered
      publish_best_detection_tf: 0
      export_best_detection: -1
      export_all_detections: 0
      debug:
        publish_stream: true
        publish_individual_tag_solution_tfs: true
        publish_group_solution_tfs: true

    filtering:
      bounds_min: [ 0.2, 0.2, -0.3 ]  # minimum valid bound from which tags should be detected (world frame)
      bounds_max: [ 6.68, 5., 0.3 ]   # maximum valid bound from which tags should be detected (world frame)
      thresh:
        min_tags_per_range: 0.6         # there must be at least this number of tags per [avg] meter of distance to the detected tags
        max_rms_per_tag: 0.40           # maximum valid RMS error per tag
        min_sum_pix_area: 7500.         # minimum total pixels that detected tags must occupy

    aruco:
      predefined_family_idx: 20     # tag family to detect -- 20 = AprilTag 36h11
      tag_ids: [ 0., 1., 2., 3., 4., 5. ]   # the tag ids which should be used, and whose corners are defined below:

      tag0_static: false
      tag0_frames: [ "base_link" ]
      tag0_corners: [ 0.230, -0.354, 0.401,
                      0.038, -0.354, 0.397,
                      0.034, -0.354, 0.589,
                      0.226, -0.354, 0.593 ]
      tag1_static: false
      tag1_frames: [ "base_link" ]
      tag1_corners: [ 0.038, 0.354, 0.397,
                      0.230, 0.354, 0.401,
                      0.226, 0.354, 0.593,
                      0.034, 0.354, 0.589 ]
      tag2_static: false
      tag2_frames: [ "base_link" ]
      tag2_corners: [ -0.003, -0.353, 0.338,
                      -0.100, -0.281, 0.336,
                      -0.102, -0.281, 0.456,
                      -0.005, -0.353, 0.458 ]
      tag3_static: false
      tag3_frames: [ "base_link" ]
      tag3_corners: [ -0.100, 0.281, 0.336,
                      -0.003, 0.353, 0.338,
                      -0.005, 0.353, 0.458,
                      -0.102, 0.281, 0.456 ]
      tag4_static: false
      tag4_frames: [ "base_link" ]
      tag4_corners: [ 0.364, -0.280, 0.477,
                      0.303, -0.360, 0.476,
                      0.301, -0.360, 0.576,
                      0.362, -0.280, 0.577 ]
      tag5_static: false
      tag5_frames: [ "base_link" ]
      tag5_corners: [ 0.303, 0.360, 0.476,
                      0.364, 0.280, 0.477,
                      0.362, 0.280, 0.577,
                      0.301, 0.360, 0.576 ]
